libraries/libfc/include/fc/crypto/public_key.hpp

```diff
@@ -1,4 +1,6 @@
 #pragma once
+#include "../../../../libfc-lite/include/fc-lite/crypto/chain_types.hpp"
```

Comment:
Would be better as `#include <fc-lite/crypto/chain_types.hpp` with the appropriate cmake change.